File: backend/src/main/java/com/runway/auth/dto/ReissueResponse.java
package com.runway.auth.dto;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class ReissueResponse {

    private String accessToken;
    private String refreshToken;
    private String tokenType;

    public static ReissueResponse of(String accessToken, String refreshToken) {
        return ReissueResponse.builder()
                .accessToken(accessToken)
                .refreshToken(refreshToken)
                .tokenType("Bearer")
                .build();
    }
}

File: android/app/src/main/java/com/runway/android/data/auth/model/ReissueResponse.kt
package com.runway.android.data.auth.model

/** POST /api/auth/reissue → ApiResponse<ReissueResponse>.data */
data class ReissueResponse(
    val accessToken: String,
    val refreshToken: String,
    val tokenType: String,
    val expiresIn: Int,
)
